docs/collections/documents/tutorials/working-with-documents.md

```diff
@@ -75,28 +75,33 @@ Let's assume your
   if __name__ == '__main__':
 
     # Variables - URLs
-    global_url = "gdn.paas.macrometa.io"
-    region_urls = [
-        "https://gdn-sfo2.prod.macrometa.io",
-        "https://gdn-us-west1.prod.macrometa.io",
-        "https://gdn-nyc1.prod.macrometa.io"
+    GLOBAL_URL = "gdn.paas.macrometa.io"
+    REGION_URLS = [
+        "gdn-us-west.paas.macrometa.io",
+        "gdn-us-east.paas.macrometa.io",
+        "gdn-us-central.paas.macrometa.io"
+        "gdn-eu-west.paas.macrometa.io",
+        "gdn-eu-central.paas.macrometa.io",
+        "gdn-ap-west.paas.macrometa.io",
+        "gdn-ap-south.paas.macrometa.io",
+        "gdn-ap-northeast.paas.macrometa.io",
+        "gdn-ap-sydney.paas.macrometa.io",
     ]
 
     # Variables - DB
-    email = "nemo@nautilus.com"
-    password = "xxxxx"
-    geo_fabric = "_system"
-    collection_name = "ddoslist"
-    ip_address = "20.1.1.9"
+    EMAIL = "nemo@nautilus.com"
+    PASSWORD = "xxxxx"
+    GEO_FABRIC = "_system"
+    COLLECTION_NAME = "ddoslist"
+    IP_ADDRESS = "20.1.1.9"
 
     # Variables - Queries
-    read_query = "FOR device in ddoslist " + "FILTER device.ip == \"" + ip_address + "\"RETURN { IP:device.ip, IsAllowed:device.action}"
-
-    insert_query = "INSERT { \"ip\" : \"" + ip_address + "\", \"action\": \"block\", \"rule\":\"blacklistA\"} INTO ddoslist"
+    READ_QUERY = f"FOR device in ddoslist FILTER device.ip == {IP_ADDRESS} RETURN" + "{IP:device.ip, IsAllowed:device.action}"
 
+    INSERT_QUERY = "INSERT { \"ip\" : \"" + IP_ADDRESS + "\", \"action\": \"block\", \"rule\":\"blacklistA\"} INTO ddoslist"
 
     # Variables - Data
-    data = [
+    DATA = [
       {"ip": "10.1.1.1", "action": "block", "rule": "blacklistA"},
```

Comment:
Could we change "blacklist" to "blocklist" without too much issue? General convention nowadays is to use allowlist and blocklist rather than whitelist and blacklist.